internal/ethapi/tracer.go

```diff
@@ -316,7 +330,22 @@ func wrapError(context string, err error) error {
 	return fmt.Errorf("%v    in server-side tracer function '%v'", message, context)
 }
 
-// CaptureState implements the Tracer interface to trace a single step of VM execution
+// CaptureState implements the Tracer interface to initialize the tracing operation.
+func (jst *JavascriptTracer) CaptureStart(from common.Address, to common.Address, create bool, input []byte, gas uint64, value *big.Int) error {
+	jst.ctx["type"] = "CALL"
```

Comment:
Maybe you should call the type `calltype`/`createtype` or something, instead of `CALL`/`CREATE`, since the opcodes `CALL`/`CREATE` is not really involved.